src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+list.querySelectorAll('li').forEach((el) => {
+  const span = document.createElement('span');
+
+  span.textContent = el.firstChild.textContent.trim();
+  el.firstChild.replaceWith(span);
+});
+
+list.addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    const listItem = e.target.closest('li');
+    const subList = listItem.querySelector('ul');
```

Comment:
The querySelector for 'ul' assumes that the sublist is a direct child of the list item. If the structure is more complex, consider using a more specific selector or checking the hierarchy.